src/components/common/Alert/Alert.tsx

```diff
@@ -1,10 +1,23 @@
 import { useCallback, useState } from 'react'
+import type { ReactNode } from 'React'
 import UIAlert from 'src/components/ui/Alert'
 import { mark } from 'src/sdk/tests/mark'
-import type { PropsWithChildren } from 'react'
 import Icon from 'src/components/ui/Icon'
 
-function Alert({ children }: PropsWithChildren<unknown>) {
+interface Props {
+  icon: string
+  text: ReactNode
+  linkText?: string
+  actionLink?: string
```

Comment:
What do you think about keeping the same pattern from the `ui/Alert` [component](https://github.com/vtex-sites/gatsby.store/blob/c8ddd48e8897c0e65d90440755f566055cca15e6/src/components/ui/Alert/Alert.tsx#L12-L15)?

```suggestion
  link?: {
    to: string
    text: string
  }
```

Reply:
That's what I tried at first, but I couldn't create the same nesting in the `sections.json` in the `storeframework.store`.  But I just did another try right now, I think I've learned how to do it. Let me try.

Reply:
It worked ✨ 
`Link Text` and `Action Link` are just labels, but internally they are `text` and `to` 
<img width="675" alt="image" src="https://user-images.githubusercontent.com/10616243/167728020-4c5b4c5c-2f79-4155-b8c5-cd6332e09b5d.png">